app/src/main/java/com/example/mylittlelibrary/ui/activities/MainActivity.kt

```diff
@@ -26,7 +26,7 @@ class MainActivity : AppCompatActivity() {
         setContentView(binding.root)
         // testing
 
-        var intent = Intent(this, ListActivity::class.java)
+        val intent = Intent(this, ListActivity::class.java)
```

Comment:
The intent variable can be declared with val instead of var because the intent is not modified after being declared, which was a suggested change made by Android Studio.